src/vmm/src/devices/virtio/queue.rs

```diff
@@ -574,8 +574,13 @@ impl Queue {
         self.next_avail -= Wrapping(1);
```

Comment:
Could you please re-word the first commit into something like:

```
feat(virtio): split `add_used()` into 2 separate methods

`Queue::add_used()` method first writes a Descriptor head in the used descriptor
ring buffer and then advances the index of this buffer to let the guest know we
used one or more Descriptors.

Carve out each one of these steps in their own function so that we can add multiple
descriptors in the used ring and advance the index only once we finish handling
descriptors in one step.

This will be useful when, in later commits, we will implement mergeable buffers for
the RX queue of the network device

Reply:
done